pkg/3scale/amp/operator/system_postgresql_options_provider.go

```diff
@@ -189,3 +189,10 @@ func (s *SystemPostgresqlOptionsProvider) podTemplateLabels() map[string]string
 
 	return labels
 }
+
+func (s *SystemPostgresqlOptionsProvider) setPriorityClassNames() {
```

Comment:
this method is never called. That's the why the postgresql pod does not get the priority class name assigned in the APIManager CR

Reply:
done. thank you